### ISSUE TITLE
Restructure Ticket Systems

### ISSUE THREAD

**Issue by Puncher1** [School-Simplified/Timmy-SchoolSimplified#279]
It's a bit messy with all the ticket systems and different files. 
Potential structure:

```
bots
└── TicketBot
    ├── ui.py                                  # for the buttons, modals, etc.  
    └── tickets                                # the system (ticket systems seperated in classes)
        ├── hr_tickets.py                      # all HR ticket systems
        ├── tech_tickets.py                    # all tech ticket systems
        ├── ...           
```

**Comment by Space-Turtle0** [comment]
**Current List of classes we need to move from `core/common.py` to `bots/TicketBot/view_classes.py`**
*Maybe not ui.py since discord.ui exists.*

Extraction list
- [ ] `TicketTempConfirm`
- [ ] `TicketLockButton`
- [ ] `NitroConfirmFake`
- [ ] `TempConfirm`
- [ ] `GSuiteVerify`
- [ ] `LockButton`
- [ ] `TechnicalCommissionConfirm`
- [ ] `EmailDropdown`
- [ ] `MGMCommissionButton`
- [ ] `StaffAnnouncements`
- [ ] `StaffApps`
- [ ] `TEmailForm`
- [ ] `IEmailForm`
- [ ] `HREmailConfirm`
- [ ] `HREmailDisabled`
- [ ] *Function* `create_ticket_button`
- [ ] *Function* `create_ui_modal_class`
- [ ] *Variable* `EmailSelectOptions`

**Comment by Space-Turtle0** [comment]
Fixed in commit 4d43832, though I'll leave it open if @Puncher1 wants to add anything.
